daemon/algod/api/server/v2/generated/model/types.go

```diff
@@ -696,6 +708,30 @@ type SimulationEvalOverrides struct {
 	MaxLogSize *uint64 `json:"max-log-size,omitempty"`
 }
 
+// SimulationOpcodeTraceUnit The set of trace information and effect from evaluating a single opcode.
+type SimulationOpcodeTraceUnit struct {
+	// Pc The program counter of the current opcode being evaluated.
+	Pc uint64 `json:"pc"`
```

Comment:
Go convention is that this should be `PC` or `pc`, not `Pc`.

Reply:
Alas. This is from generated code, and same happens to `DryrunState`...

We can get around this `Pc` generated code if only we decide to write `"PC"` rather than `"pc"` in oas2 file.

Reply:
And that, in turn, would make the json field "PC"?  Which is is pretty ugly, so I suppose this is fine. I'm surprised I even noticed, I usually skip generated code.

Reply:
yes, if only you *specify* in oas2 json object that this field name is "PC".